### PR TITLE
Adds banarnarbread

### PR DESCRIPTION
**Pull request by iantine**
## What Does This PR Do
Adds banarnarbread. A play on banana bread with a Nar'Sie theme!

## Why It's Good For The Game
A cult themed food can be fun for chefs to play with. The recipe also involves blood, which is a rarity for chefs to get to use in their cooking.

Open to suggestions on tweaks and changes!

Recipe:
prv2
Sprites:
prv1

## Changelog
:cl:
add: adds banarnarbread
/:cl:

### CODE REVIEW

**Review by S34NW — APPROVED** [review]
Grammar looks good, I love this :)

**Comment by Darkmight9** [comment]
Deliciously heretical 👌

File: code/modules/food_and_drinks/food/foods/bread.dm [hunk 1]
```diff
@@ -126,6 +126,25 @@
...
 	tastes = list("bread" = 10, "cheese" = 10)
 
+/obj/item/reagent_containers/food/snacks/sliceable/banarnarbread
+	name = "banarnarbread loaf"
+	desc = "A loaf of delicious mah'weyh pleggh at e'ntrath!"
+	icon_state = "banarnarbread"
+	slice_path = /obj/item/reagent_containers/food/snacks/banarnarbreadslice
+	slices_num = 5
+	filling_color = "#6F0000"
+	list_reagents = list("nutriment" = 15, "vitamin" = 5)
```
Comment (dearmochi) [inline code/modules/food_and_drinks/food/foods/bread.dm:136]: Is it intended that eating the loaf whole gives less nutriments than the sum of all slices would?

Comment (iantine) [reply code/modules/food_and_drinks/food/foods/bread.dm:136]: Nope, I'm just bad at maths lol
Fixing now!

**Review by AffectedArc07 — APPROVED** [review]

**Comment by PacifistDalek** [comment]
@iantine hey there, couldn't find you on the discord, but I was wondering if you'd have any thoughts or insights as I'm currently respriting our breads to match TG's bread sprites! 😊

**Pull request merged**